crates/bevy_render/src/render_graph/nodes/window_texture_node.rs

```diff
@@ -62,15 +64,12 @@ impl Node for WindowTextureNode {
                 .iter(&window_resized_events)
                 .any(|e| e.id == window.id())
         {
-            let render_resource_context = render_context.resources_mut();
-            if let Some(RenderResourceId::Texture(old_texture)) = output.get(WINDOW_TEXTURE) {
-                render_resource_context.remove_texture(old_texture);
-            }
+            // Update TextureNode descriptor
+            self.inner.texture_descriptor_mut().size.width = window.physical_width();
+            self.inner.texture_descriptor_mut().size.height = window.physical_height();
```

Comment:
Since `texture_descriptor_mut` calls `set_changed` internally, it probably worth changing these lines to:

```rust
let size = self.inner.texture_descriptor_mut().size;
size.width = window.physical_width();
size.height = window.physics__height();
```

This then doesn't require us to unnecessarily call `set_changed` more than once.

Reply:
Done (albeit with a temporary for texture_descriptor).